include/temperature_correction.hpp

```diff
@@ -0,0 +1,126 @@
+#include "WaveformFitResult.hpp"
```

Comment:
Non-accelerometer related? Should include such changes in a different PR, or include in the title/description of this one.

Reply:
No significant change, just a test I was doing most probably

Reply:
OK, please remove anything unnecessary to the specific feature you're implementing in this PR. In general, you want to minimize the amount of code in each PR to facilitate review and debugging.